src/transformers/__init__.py

```diff
@@ -4034,6 +4007,7 @@
         from .models.poolformer import PoolFormerFeatureExtractor, PoolFormerImageProcessor
         from .models.segformer import SegformerFeatureExtractor, SegformerImageProcessor
         from .models.swin2sr import Swin2SRImageProcessor
+        from .models.tvlt import TvltImageProcessor, TvltProcessor
```

Comment:
```suggestion
        from .models.tvlt import TvltImageProcessor
```
Same comment here, the processor itself should be importable just like `TvltConfig` (i.e. no dependency)